dom-event-delegation/dom-event-delegation-quiz-notes.md

```diff
@@ -0,0 +1,52 @@
+# dom-event-delegation-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the `event.target`?
+the element you are adding the event listener to;
```

Comment:
Not quite! `event.target` is the element that was interacted with to trigger the event. `event.currentTarget` would be the element that has the event listener attached to it.